azuremetadata

```diff
@@ -49,6 +54,13 @@ try:
         util.print_help()
         exit()
 
+    if args.show_apis:
```

Comment:
Name change ;) `show_apis` -> `listapis`

Reply:
Done, when all is ok, I will squash them in one commit.